### PR TITLE
Set up test Coverage Metrics

### PR DESCRIPTION
**Pull request by pedbr**
- Installs Coveralls Gem
- Adds Coveralls configurations

PT: https://www.pivotaltracker.com/story/show/167285250

### CODE REVIEW

File: coverage/.last_run.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (oliverochman) [inline coverage/.last_run.json]: you need to add the coverage folder to the `.gitignore`

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -5,6 +5,8 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'coveralls'
```
Comment (oliverochman) [inline spec/rails_helper.rb]: these two lines need to be on top of the file

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: coverage/.last_run.json (hunk 1 above)
Comment (pedbr) [reply coverage/.last_run.json]: added!

File: spec/rails_helper.rb (hunk 2 above)
Comment (pedbr) [reply spec/rails_helper.rb]: done!

**Pull request closed without merging**